src/microbe_stage/CompoundBag.cs

```diff
@@ -12,21 +12,20 @@ public class CompoundBag : ICompoundStorage
 {
     private readonly HashSet<Compound> usefulCompounds = new();
 
-    /// <summary>
-    ///   Creates a new bag
-    /// </summary>
-    /// <param name="capacity">Specifies the initial capacity of the compound bag</param>
-    public CompoundBag(float capacity)
+    [JsonProperty]
+    private Dictionary<Compound, float>? compoundCapacities;
+
+    public CompoundBag(float nominalCapacity)
     {
-        Capacity = capacity;
+        NominalCapacity = nominalCapacity;
     }
 
     /// <summary>
-    ///   How much of each compound this bag can store.
-    ///   Currently a CompoundBag can hold the same amount of each compound.
+    ///   Stores the default capacity for all compounds that do
```

Comment:
```suggestion
    ///   Specifies the default capacity for all compounds that do
```
As this variable doesn't "store" the data just specifies how much something can be stored elsewhere.